item-impl/src/test/java/com/example/auction/item/impl/ItemEntityTest.java

```diff
@@ -274,7 +274,7 @@ public void shouldReturnItemWithWinningBidAfterAuction() {
 
     //  ---------------------------------------------------------------------------------------------------------
 
-    private <T> void expectEvents(Outcome<T, ?> outcome, T... expected) {
+    private void expectEvents(Outcome<?, ?> outcome, Object... expected) {
```

Comment:
Type parameters and varargs do not combine well in Java, and the original code here produces a warning. See http://www.angelikalanger.com/GenericsFAQ/FAQSections/ProgrammingIdioms.html#FAQ300

In this case, we don't actually care about the type of the expected events, since we are using them in `equals`.